src/generic/ConfirmationPopup.jsx

```diff
@@ -9,6 +9,9 @@ const ConfirmationPopup = ({
   confirmLabel,
   onCancel,
   cancelLabel,
+  confirmVariant,
+  confirmButtonClass,
+  cancelButtonClass,
```

Comment:
these props are affecting class of confirmation pop up in <InContextDiscussionFields/> 
please review